pkgs/tools/misc/zf/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
```

Comment:
oh fuck, another flamewar about pretty formatters

Reply:
I swear https://github.com/NixOS/rfcs/pull/101 can come soon enough.

Reply:
I can’t wait.

Reply:
Yes, alejandra sucks. It is very inconsistent with its won whitespace by enforcing new lines and extra space at one place and at others stripping every last space out of code.